governance.md

```diff
@@ -0,0 +1,60 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) develops and deploys open source software for open access publishing and enables publication credit for software and other research and educational products. It is a fiscally sponsored project of the 501(c)(3) [NumFOCUS Foundation](https://www.numfocus.org).
```

Comment:
Can we add something about coordinating the operations of the Open Journals, or something similar?